config/release-45x-211.conf

```diff
@@ -0,0 +1,84 @@
+##### Defined in the script #####
+# CURRENT_DIR: execution dir
+#####
+
+##### local dirs #####
+# Unlikely to need modification
+#ZINC_DIR=${CURRENT_DIR}/zinc
+#SCALATEST_DIR=${SCRIPT_DIR}/scalatest
+#BUILD_DIR=${CURRENT_DIR}/target
+#LOCAL_M2_REPO=${BUILD_DIR}/m2repo
+IDE_M2_REPO=${LOCAL_M2_REPO}
```

Comment:
`IDE_M2_REPO` can be removed, it is no longer needed. I would say that you should remove the entire `local dirs` sections, since these values are just commented out.